frontend/apps/marketing/src/types/contentful/entries/SeoMetadata.ts

```diff
@@ -0,0 +1,17 @@
+import {EntrySkeletonType} from 'contentful';
+
+import {ExperienceAsset} from '@/types/contentful/ExperienceAsset';
+
+export type SeoMetadata = {
+  seoTitle: string;
+  seoDescription: string;
+  hidePageFromSearchEnginesNoindex: boolean;
+  hideLinksFromSearchEnginesNofollow: boolean;
+  keywords: string[];
+  canonicalUrl: string;
+  openGraphTitle: string;
+  openGraphDescription: string;
+  openGraphImage: ExperienceAsset;
+};
+
+export type SeoMetadataEntry = EntrySkeletonType<SeoMetadata>;
```

Comment:
I don't have full context on usage of this Entry, but if it follows general Contentful approach where when you receive entry you receive it with `sys` and `metada` attributes, I think we should continue typing them as well even if it's not used right now. (I think we should use `ContntfulEntry` generic type here as well)

Reply:
Yes, good catch. I've updated it to use the new `Entry` type (which does the same as what `ContentfulEntry` used to do but uses the appropriate contentful type guards)